ocmsrnaseq/pipeline_fastqc.py

```diff
@@ -110,8 +111,116 @@ def build_report(infiles, outfile):
     statement = '''multiqc . -s'''
     P.run(statement)
 
-@follows(build_report)
```

Comment:
You can also make the output directory here instead of explicitly in the function below i.e. ```@follows(build_report, mkdir("multiqc_data/failed_qc_metrics"))```